components/Settings.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react'
-import enhanceWithClickOutside from 'react-click-outside'
 import omitBy from 'lodash.omitby'
 
 import ThemeSelect from './ThemeSelect'
 import FontSelect from './FontSelect'
 import Slider from './Slider'
 import Toggle from './Toggle'
-import WindowPointer from './WindowPointer'
+import Popout from './Popout'
+import Button from './Button'
 import { COLORS, DEFAULT_PRESETS } from '../lib/constants'
-import { getPresets, savePresets } from '../lib/util'
```

Comment:
Great catch! Is there an eslint rule to prevent duplicate imports like this?

Reply:
https://eslint.org/docs/rules/no-duplicate-imports

We could add that to our eslint, surprised it's not in the recommended rules.

Reply:
Please add it 👍